gadgets/trace_exec/program.bpf.c

```diff
@@ -249,6 +257,8 @@ int BPF_KPROBE(security_bprm_check, struct linux_binprm *bprm)
 	u32 pid = (u32)bpf_get_current_pid_tgid();
 	struct event *event;
 	char *file;
+	unsigned int dev_no;
```

Comment:
This one needs to be `dev_t`, since it comes from https://github.com/torvalds/linux/blob/v6.8/include/linux/fs.h#L1200
```suggestion
	dev_t dev_no;
```

(I was suggesting to use `unsigned int` only for `dev_major` and `dev_minor` in `struct event`)